test/framework/twilio-video-react/src/App.js

```diff
@@ -7,13 +7,23 @@ export default class App extends Component {
 
     this.state = {};
 
-    connect(this.props.token).then(room => {
-      this.setState({ room });
-      room.once('disconnected', () => this.forceUpdate());
-      room.disconnect();
-    }, error => {
-      this.setState({ error });
-    });
+    if (this.props.environment) {
+      connect(this.props.token, Object.assign({ environment: this.props.environment })).then(room => {
+        this.setState({ room });
```

Comment:
instead of duplicating this code block, I would just create connectOptions outside like
```
const connectOptions =  this.props.environment ? { environment: this.props.environment } : {};
connect(this.prop.token, connectOptions).then(room => {
...
```